docs/DEVELOPER.md

```diff
@@ -3,7 +3,7 @@
 ## What is TestSys
 
 TestSys is a framework that leverages Kubernetes (K8s) to create resources and run tests.
-To learn more about how it works, see the [design](design/DESIGN.md) document.
+To learn more about how it works, see the [design](docs/DESIGN.md) document.
```

Comment:
This doc is already in the `docs` folder, so I think you need to drop that from the path so it doesn't try to go to `docs/docs/DESIGN.md`.

Reply:
Ah darn! 🥇